src/site/content/en/blog/link-prefetch/index.md

```diff
@@ -45,6 +47,10 @@ For example, in a product listing page, you can prefetch the page for the most p
 eBay implemented prefetching for the first five results on a search page to speed up future pages loads and saw a positive impact on conversion rates.
 {% endAside %}
 
+Prefetching entire pages—while costly in terms of bandwidth—can greatly improve most performance metrics. [Time to First Byte (TTFB)](/ttfb/) will often be much lower, as the document request results in a cache hit. Because TTFB will be much lower, any other subsequent time-based metrics will often be much faster, including [Largest Contentful Paint (LCP)](/lcp/) and [First Contentful Paint (FCP)](/fcp/).
```

Comment:
I'm not sure about the wording "entire pages". `<link rel="prefetch">` only fetches the resource requested (e.g. the document, or a subresource) not the entire page. `<link rel="prerender">` currently fetches a page and all its sub resources, but that's changing, is less well supported, and is out of scope for this article.

Also "while costly in terms of bandwidth" is a bit drastic.

Maybe something like this:

```suggestion
While prefetching resources does use additional bandwidth, it can greatly improve most performance metrics. [Time to First Byte (TTFB)](/ttfb/) will often be much lower, as the document request results in a cache hit. Because TTFB will be much lower, any other subsequent time-based metrics will often be much faster, including [Largest Contentful Paint (LCP)](/lcp/) and [First Contentful Paint (FCP)](/fcp/).
```